litestar/contrib/pydantic/pydantic_dto_factory.py

```diff
@@ -37,6 +37,8 @@
     OpenAPIType.STRING: str,
     OpenAPIType.INTEGER: int,
     OpenAPIType.BOOLEAN: bool,
+    OpenAPIType.OBJECT: str,
+    OpenAPIType.NUMBER: float,
```

Comment:
These are the openapi types that are in the `PYDANTIC_TYPE_MAP` that we didn't have down types for.